pir/common/shard.go

```diff
@@ -14,3 +14,10 @@ type Shard interface {
 	GetData() []byte
 	Read(reqs []byte, reqLength int) ([]byte, error)
 }
+
+// PIRBackings is a static table of the registered / available PIR implementations
+var PIRBackings map[string]func(int, []byte, string) Shard
+
+func init() {
```

Comment:
You shouldn't be exporting a data structure. Can we export a set of functions that registers/removes/searches for the proper thing?